client/screens/ProfilePage.js

```diff
@@ -39,6 +42,23 @@ const ProfilePage = (props) => {
   const userRef = ref(db, "users/" + props.userId);
   const newUserRef = push(userRef);
 
+  // Image Picker
+  const pickImage = async () => {
+    // No permissions request is necessary for launching the image library
+    let result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.All,
+      allowsEditing: true,
+      aspect: [4, 3],
+      quality: 1,
+    });
+
+    console.log(result);
```

Comment:
Leftover console log here, let's be sure to remove this on the next PR.

Reply:
thanks for the catch, will do!